test/kubemark/start-kubemark.sh

```diff
@@ -366,23 +346,4 @@ echo -e "Getting apiserver data partition status:" >&2
 echo
 echo -e "Getting ETCD data partition status:" >&2
 "${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get etcd
-echo
-
-### for multiple tenant tests, set up the tenants on desired TP clusters
```

Comment:
same question here. it's not obvious from the PR why these are removed .

Reply:
These code created two test tenants, arktos/zeta, when we were trying with 2 TPs. Yet the code was changed to create variable number of TPs, where two test tenants may not be enough. We need the tester to create the test tenants on the fly. Therefore this section of code is removed.

Reply:
thanks! consider this comment resolved.